renku/service/serializers/cache.py

```diff
@@ -210,20 +207,30 @@ class ProjectListResponseRPC(JsonRPCResponse):
     result = fields.Nested(ProjectListResponse)
 
 
-class ProjectMigrateRequest(RepositoryContext):
+class ProjectMigrateRequest(AsyncSchema, CommitSchema, LocalRepositorySchema, RemoteRepositorySchema):
     """Request schema for project migrate."""
 
     force_template_update = fields.Boolean(default=False)
     skip_template_update = fields.Boolean(default=False)
     skip_docker_update = fields.Boolean(default=False)
     skip_migrations = fields.Boolean(default=False)
 
+    # ! temporary -- should be removed but there are tests to update
+    migrate_project = fields.Boolean(default=False, missing=False)
+
     @pre_load()
     def default_commit_message(self, data, **kwargs):
         """Set default commit message."""
+        # ? Should we move this to MigrateProjectCtrl?
```

Comment:
I think this makes sense, I guess you'll do it in your new PR for commit messages.